store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -39,11 +39,15 @@
 
     private final int mappedFileSize;
 
-    private final CopyOnWriteArrayList<MappedFile> mappedFiles = new CopyOnWriteArrayList<MappedFile>();
+    private final CopyOnWriteArrayList<MappedFile> mappedFiles = new CopyOnWriteArrayList<>();
 
     private final AllocateMappedFileService allocateMappedFileService;
 
-    private long flushedWhere = 0;
```

Comment:
yes, I understand this, but `flushedWhere` doesn't sound good. File's offset is the file's offset, but this offset is the offset withing the queue of all files ;) Should we rename it to `flushedPosition`? @zhouxinyu What do you think?